contribution_to_project.rst

```diff
@@ -0,0 +1,95 @@
+***********************
+Contributing to Ansible
+***********************
+
+This document describes ways how you can contribute to the Open Source projects of the Ansible ecosystem.
+
+.. contents:: Topics
+
+What is a contribution to an Open Source project
+================================================
+
+A contribution to an Open Source project is any good given by an individual or a company to the project including increasing its functionality, improvements of the codebase, code developer experience, user experience, increasing the project's popularity and community, providing infrastructure, and any other kinds of active social or material support.
+
+Most the types of contribution to an Open Source project do not require programming skills.
```

Comment:
I'm speaking about the things mentioned above, so that's why i use `the`. But i think the suggested generalized form w/o `the` should also work. Thanks, committed